actionmailbox/lib/action_mailbox/errors.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "active_support/actionable_error"
+require "rails/command"
+
+module ActionMailbox
+  # Generic base class for all Action Mailbox exceptions.
+  class Error < StandardError; end
+
+  # Raised when we detect that Action Mailbox has not been initialized.
+  class SetupError < Error
```

Comment:
How about `MissingInstallError`? It's not like there was a problem in setting up the library.

Reply:
Yeah, I was just thinking about that. Maybe just `InstallError`?